### ISSUE TITLE
Still not fixed: Fix Helm Charts is not downloadable via UI #15228 ?

### ISSUE THREAD

**Issue by CheaterAdams** [goharbor/harbor#15737]
This is my first post, so please forgive me if I need to take this discussion elsewhere.
I have upgraded Harbor from v2.3.2 to v2.3.3 and am still seeing the issue reported #15228. 

![image](https://user-images.githubusercontent.com/92112704/136439336-91fe9caa-20d4-426d-be3e-910ba3f3fa51.png)

I installed from the offline installer (https://github.com/goharbor/harbor/releases) for v2.3.3 using docker-compose.

root@harbor [ ~/harbor ]# docker-compose images
Container Repository Tag Image Id Size

chartmuseum goharbor/chartmuseum-photon v2.3.3 24b85ee1f3ff 178.8 MB
harbor-core goharbor/harbor-core v2.3.3 f065d61635a8 192.6 MB
harbor-db goharbor/harbor-db v2.3.3 7f32a2a21263 236.9 MB
harbor-jobservice goharbor/harbor-jobservice v2.3.3 cb3b2c9d8f57 210.6 MB
harbor-log goharbor/harbor-log v2.3.3 77b2d1cb3ab2 168.2 MB
harbor-portal goharbor/harbor-portal v2.3.3 eeb24c004c47 58.2 MB
nginx goharbor/nginx-photon v2.3.3 43487013ae1e 44.98 MB
redis goharbor/redis-photon v2.3.3 5b8e952b8f45 164.6 MB
registry goharbor/registry-photon v2.3.3 08b0e6fe666e 81.88 MB
registryctl goharbor/harbor-registryctl v2.3.3 ffea0a2c3674 132.7 MB
trivy-adapter goharbor/trivy-adapter-photon v2.3.3 271525f11619 130.1 MB  

From the UI About...

![image](https://user-images.githubusercontent.com/92112704/136439379-17968dff-82a5-45de-ad77-47a049fb88f4.png)

Docker version: 18.06.1
Docker-compose:  1.29.2, build 5becea4c

Is there something I am missing. How can I be sure I have the fixed code?
Any help would be appreciated.
Thanks

**Comment by AllForNothing** [comment]
@CheaterAdams What about clearing your browser cache, and try again?

**Comment by CheaterAdams** [comment]
I assume that resolved the issue. Yesterday I taken down the vm where helm was installed and from where I launched the browser in preparation for other activities. This morning after seeing your comment I launched the vm to test and the issue was not seen. I guess that essentially cleared the browser cache.

Browser cache has bitten me on other occasions. I should have thought of it.

Thank you so much for the help.
